tests/multi_step/test_correctness.py

```diff
@@ -22,9 +23,29 @@
     "16",
 ]
 
+NUM_LOGPROBS = [None, 5]  # `logprobs` argument to OpenAI completions API
```

Comment:
Seems like this is only used once. In this case we could just embed it.

Reply:
Done.